.github/workflows/ci.yml

```
@@ -64,14 +64,41 @@ jobs:
           python-version: ${{ matrix.python-version }}
           allow-prereleases: true
 
-      - name: Test typing_extensions
+      - name: Install coverage
+        if: ${{ !startsWith(matrix.python-version, 'pypy') }}
+        run: |
+          # Be wary that this does not install typing_extensions in the future
+          pip install coverage
+
+      - name: Test typing_extensions with coverage
+        if: ${{ !startsWith(matrix.python-version, 'pypy') }}
+        run: |
+          # Be wary of running `pip install` here, since it becomes easy for us to
+          # accidentally pick up typing_extensions as installed by a dependency
+          cd src
+          python --version  # just to make sure we're running the right one
+          export COVERAGE_FILE=.coverage_${{ matrix.python-version }}
+          # Run with coverage omit files that are executed in tempfiles
+          coverage run --omit ann_module*,inspect* -m unittest test_typing_extensions.py
```

nit: I'd do this, otherwise the `python --version` command we ran immediately above doesn't have much value :-)

```suggestion
          python -m coverage run --omit ann_module*,inspect* -m unittest test_typing_extensions.py
```